evergreen/.flake8

```diff
@@ -2,3 +2,4 @@
 max-line-length = 88
 extend-ignore = E203,E701
 exclude = .git,__pycache__,.venv,venv
```

Comment:
You can exclude `./evergreen/staticfiles/admin` if you want a higher standard for us without annoyances from Django

Reply:
done